EIPS/eip-4671.md

```diff
@@ -0,0 +1,468 @@
+discussions-to: https://ethereum-magicians.org/t/eip-4671-non-tradable-token/7976?u=omaraflak
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-01-13
+requires: 165
+---
+
+# Non-Tradable Token Standard
+
+<!-- AUTO-GENERATED-CONTENT:START (TOC) -->
+- [Simple Summary](#simple-summary)
+- [Abstract](#abstract)
+- [Motivation](#motivation)
+- [Specification](#specification)
+  - [Extensions](#extensions)
+    - [Metadata](#metadata)
+    - [Delegation](#delegation)
+- [Rationale](#rationale)
+  - [On-chain vs Off-chain](#on-chain-vs-off-chain)
+- [Implementation](#implementation)
+  - [NTT](#ntt)
+  - [NTTDelegate](#nttdelegate)
+- [NTT for EIP ?](#ntt-for-eip-)
+- [Copyright](#copyright)
+<!-- AUTO-GENERATED-CONTENT:END -->
+
+## Simple Summary
+
+A standard interface for <u>**non-tradable tokens**</u>, aka <u>**NTT**</u>s.
+
+## Abstract
+
+NTTs represent inherently personal possessions (material or immaterial), such as university diploma, online training certificates, government issued documents (national id, driving licence, visa, wedding, etc.), badges, labels, and so on.
+
+As the name implies, NTTs are not made to be traded or transfered. They don't have monetary value. They are personally delivered to **you**, and they only serve as a **proof of possession**.
+
+## Motivation
+
+US, 2017, MIT published 111 diplomas on a blockchain. France, 2018, Carrefour multinational retail corporation used blockchain technology to certify the provenance of its chickens. South Korea, 2019, the state published 1 million driving licences on a blockchain-powered platform.
+
+Each of them made their own smart contracts, with different implementations. We think diplomas, food labels, or driving licences are just a subset of a more general type of tokens: **non-tradable tokens**. Tokens that represent certificates or labels that were granted to you by some authority.
+
+By providing a common interface for this type of tokens, we allow more applications to be developed and we position blockchain technology as a standard gateway for verification of personal possessions.
+
+## Specification
+
+A single NTT contract, is seen as representing one type of badge by one authority. For instance, one NTT contract for PSN achievements, another for Ethereum EIP creators, and so on...
+
+* An address might possess multiple tokens, which are indexed.
+* An authority who delivers a certificate should be in position to invalidate it. Think of driving licences or weddings. However, it cannot delete your token.
+* The issuer of a token might be someone else than the contract creator.
+
+<!-- AUTO-GENERATED-CONTENT:START (CODE:syntax=solidity&src=./contracts/INTT.sol) -->
+<!-- The below code snippet is automatically added from ./contracts/INTT.sol -->
+```solidity
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts/utils/introspection/IERC165.sol";
```

Comment:
```suggestion
import "./IERC165.sol";
```
I think it best not to reference specific implementations, and have seen other standards use a pseudocode implementation like the one I've suggested to represent inheritance (for example, [ERC-721](./eip-721.md)).